src/app/ApplicationTemplate.Presentation/Configuration/AppServicesConfiguration.cs

```diff
@@ -24,7 +23,7 @@ public static IServiceCollection AddAppServices(this IServiceCollection services
 	{
 		return services
 			.AddSingleton<IMessageDialogService, AcceptOrDefaultMessageDialogService>()
-			.AddSingleton<IConnectivityProvider, MockedConnectivityProvider>()
+			.AddSingleton<IConnectivityRepository, MockedConnectivityRepository>()
```

Comment:
Could we move this to the Access layer configuration?

Reply:
No because it doesn't know the content of the View Layer. All view implementation are done there.

Reply:
Unless you mean the Mock one?

Reply:
Well yes this is the mock one.